Orange/widgets/visualize/owmosaic.py

```diff
@@ -447,13 +445,19 @@ def updateGraph(self, data=-1, subsetData=-1, attrList=-1, **args):
         if data == -1:
             data = self.data
 
+        if data != None:
```

Comment:
Use `is not` instead of `!=` for comparisons with singletons (see https://www.python.org/dev/peps/pep-0008/#programming-recommendations).